lib/galaxy/datatypes/binary.py

```diff
@@ -71,7 +95,7 @@ def sniff(self, filename):
     
     def set_peek(self, dataset, is_multi_byte=False):
         if not dataset.dataset.purged:
-            dataset.peek = "Binary audio file"
+            dataset.peek = "AMP audio file"
```

Comment:
Do we want to prefix these with AMP?  It's a generic classification (Audio) that isn't necessarily AMP specific.

Reply:
Good point. I put AMP to all types we extended, but if we are to contribute them, it's better to have them as generic (video/audio/wav)